cmake/kconfig.cmake

```diff
@@ -22,6 +22,7 @@ set(kconfig_target_list
   config
   gconfig
   menuconfig
+  pymenuconfig
```

Comment:
What about 'menu' ?

Reply:
My plan was to rename it to 'menuconfig' later, once we get rid of the C tools. That's what people are used to.

Reply:
Hmm, this would be incosistent with the other Kconfig-frontends I guess ...

Reply:
I agree with renaming to `menuconfig` later

Reply:
Let's not overcomplicate things. Let's replace `mconf` with `pymenuconfig` in this PR, and keep the "menuconfig" name